src/js/tabs/pid_tuning.js

```diff
@@ -32,8 +32,9 @@ TABS.pid_tuning.initialize = function (callback) {
         self.activeSubtab = 'pid';
     }
 
-    // Update filtering defaults based on API version
+    // Update filtering and pid defaults based on API version
     var FILTER_DEFAULT = FC.getFilterDefaults();
+    var PID_DEFAULT = FC.getPidDefaults();
```

Comment:
Is this being used somewhere?

Reply:
Should prolly be `this.PID_DEFAULT`.

Reply:
It is called in tuning-sliders.js and needed at the init to coordinate new values with sliders part. I didnt found a better way to do this

Reply:
@Asizon: No. `TuningSliders.PID_DEFAULT` is called in `TuningSliders.js` - what you are doing won't work.

Reply:
oh right, without this im getting sliders disabled at init :( :

![image](https://user-images.githubusercontent.com/43983086/78562285-64dde280-7819-11ea-89c1-1d6223e247d3.png)

Reply:
Ok, maybe just need to call FC.getPidDefaults here?